tools/uart_hw_test/README.md

```diff
@@ -22,13 +22,13 @@ Each of the above tests are performed for the following baud rates.
 115200, 38400, 19200, 9600, 4800
 
 
-### How to USE
+### How to USE for Linux
 1. Find the UART port to be tested. For example, on the RTK8721 board, we will perform uart test for the uart1.
 2. Connect the uart (Rx, Tx, Gnd) lines from the test device to the Linux PC using Serial-USB converter.
 3. Enable CONFIG_UART_HW_TEST config. Build and download the binary.
 4. After boot up, run the uart_hw_test command at the tash prompt
 5. Input the port number on the device and on the PC. In our example, the uart1 appears as /dev/ttyS2 on the device. On the PC, the Serial-USB converter appears as /dev/ttyUSB2. Hence the port number is 2 in both the cases.
-6. Now run the uart test tool on the PC (./uart_test.sh) and input the port number.
+6. Now run the uart test tool on the PC (./uart_test.sh) and input the port number. (Please install gcc if your system doesnot have one , because it is a prerequisite)
```

Comment:
Minor: "doesnot" -> "does not"
", because" -> "because"
"prerequisite" -> "pre-requisite"

Reply:
done